src/Client/Pages/Identity/Account.razor

```diff
@@ -4,12 +4,15 @@
  @if (!AccountManagementDisabled)
 {
     <MudTabs Elevation="25" Color="Color.Transparent" Rounded="true" PanelClass="mt-6">
-    <MudTabPanel Text="@localizer["Profile"]">
-        <Profile />
-    </MudTabPanel>
+        <MudTabPanel Text="@localizer["Profile"]">
+            <Profile />
+        </MudTabPanel>
         <MudTabPanel Text="@localizer["Security"]">
             <Security />
         </MudTabPanel>
+        <MudTabPanel Text="@localizer["Forgot Password"]">
+            <ForgotPassword />
+        </MudTabPanel>
```

Comment:
Again, I think it would be a bad idea to place the ForgotPassword Component here. Like how would you even get to this page if you already forgot your password :p

It should be ideally be accessible from the Login page. I think that's the standard way. If you think the tenant header is the issue, you can follow the same approach we have used in our login component.

Reply:
I agree with what you wrote